### DISCUSSION TITLE
Compile Python on Termux

### DISCUSSION THREAD

**Discussion by babaric-dev** [termux/termux-packages#12047] [Q&A]
How can I compile Python and generate a `.deb` (on Termux) without using build scripts from this repo? It installs a lot of stuff.

Btw, I am trying to compile a different version of Python. So `pkg install python` won't help.

**Comment by T-Dynamos** [comment]
> Btw, I am trying to compile a different version of Python. So pkg install python won't help.

Can you specify the version?

**Comment by 2096779623** [comment]
https://github.com/termux/termux-packages/wiki/Build-environment

https://github.com/Termux-pod/termux-pod
